src/modules/navigator/rtl.cpp

```diff
@@ -228,7 +228,9 @@ void RTL::find_RTL_destination()
 		}
 	}
 
-	if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING) {
+	if (_param_rtl_cone_half_angle_deg.get() > 0.0
```

Comment:
```suggestion
	if (_param_rtl_cone_half_angle_deg.get() > 0
```

Reply:
It's an int.

Reply:
And you could then remove the if(cone_half_angle_deg > 0.0f) from inside the calculate_return_alt_from_cone_half_angle() function as well, as it now won't every enter there.